auth/tests/unit/cognito-client-oidc.unit.test.ts

```diff
@@ -0,0 +1,64 @@
+import {Template} from "aws-cdk-lib/assertions";
+import {schema} from "yaml-cfn";
+import {describe, it, beforeAll} from "vitest";
+import {readFileSync} from "fs";
+import {load} from "js-yaml";
```

Comment:
Formatting

```suggestion
import { Template } from "aws-cdk-lib/assertions";
import { schema } from "yaml-cfn";
import { describe, it, beforeAll } from "vitest";
import { readFileSync } from "fs";
import { load } from "js-yaml";
```

If you run `Option + shift + f` it should auto-format an entire file in vscode (until we setup a formatter)

Reply:
Strange I corrected before committing, I think. I'll have a second look.